openfpga_flow/openfpga_shell_scripts/write_full_testbench_no_clk_modeling_example_script.openfpga

```diff
@@ -0,0 +1,75 @@
+# Run VPR for the 'and' design
```

Comment:
Depending on the test, you want to run, you may not need a new script. 
If you just want to test bitstream generation, you can reuse this script: https://github.com/lnis-uofu/OpenFPGA/blob/master/openfpga_flow/openfpga_shell_scripts/generate_bitstream_global_tile_multiclock_example_script.openfpga

On the other side, the VexRiscV and picorv benchmarks are too big to be tested on CI (iVerilog may takes hours to finish). I suggest just testing bitstream on CI.

Also to avoid adding new script, you may also consider to add options to the script which can be customized in task configuration file.
For example, 

https://github.com/lnis-uofu/OpenFPGA/blob/223e06d23c070b4d704a4e120b53e85d738cd162/openfpga_flow/openfpga_shell_scripts/write_full_testbench_example_script.openfpga#L3

You can define the variable ${OPENFPGA_<your_own_variable_name>}

In task configuration file, you can define the content of the variable:

https://github.com/lnis-uofu/OpenFPGA/blob/223e06d23c070b4d704a4e120b53e85d738cd162/openfpga_flow/tasks/basic_tests/full_testbench/configuration_chain/config/task.conf#L22

Reply:
I see what you're saying about avoiding adding a new script, but how would you disable the '--clock_modeling route'  option of vpr with a variable?

Reply:
yes. Refer to the use of ${OPENFPGA_VPR_DEVICE_LAYOUT}

Reply:
Ok, also I needed to remove the '--activity_file ${ACTIVITY_FILE}' part of one of the 'link_openfpga_arch' line as well. Since that is not determined by the config, what would be the best way to resolve that besides creating a new script?